src/pages/commits/ActiveCommits/ChartToggle.tsx

```diff
@@ -2,17 +2,24 @@ import { ClassNames } from "@emotion/react";
 import styled from "@emotion/styled";
 import { uiColors } from "@leafygreen-ui/palette";
 import { RadioGroup, Radio } from "@leafygreen-ui/radio-group";
-import { Label } from "@leafygreen-ui/typography";
 import { useProjectHealthAnalytics } from "analytics/projectHealth/useProjectHealthAnalytics";
+import { Accordion } from "components/Accordion";
 import { size } from "constants/tokens";
 import { ChartTypes } from "types/commits";
 
 const { gray } = uiColors;
 
 export const ChartToggle: React.VFC<{
+  onToggleAccordion: (nextState: { isVisible: boolean }) => void;
+  defaultOpenAccordion: boolean;
```

Comment:
Do we need to pass these props in? i think the state logic for this component would make sense living in this file. Since it is only used for the chart.